packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -50,14 +51,20 @@ import { ISocketStorageDiscovery } from "./contractsPublic";
  * This constructor should be used by environments that support dynamic imports and that wish
  * to leverage code splitting as a means to keep bundles as small as possible.
  */
-export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
+export class OdspDocumentServiceFactoryCore
+	implements IDocumentServiceFactory, ISessionAwareDriverFactory
```

Comment:
Rename ISessionAwareDriverFactory -> IRelayServiceSessionAwareDriverFactory

Reply:
Just out of curiosity, what is the relay service exactly?

Reply:
Push session. We refer to it as relay service. "Session" could be of anything.